en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md

```diff
@@ -272,27 +272,6 @@ list of items and indirectly calling `forTemplate` using the [`$Me` template var
 This method will be used by the `cmsPreview` action in the `MyAdmin` class to tell the
 CMS what to display in the preview panel.
 
-The `forTemplate` method will probably look something like this:
-
-```php
-namespace App\Model;
-
-use SilverStripe\ORM\CMSPreviewable;
-use SilverStripe\ORM\DataObject;
-
-class Product extends DataObject implements CMSPreviewable
-{
-    // ...
-
-    public function forTemplate(): string
-    {
-        // If the template for this DataObject is not an "Include" template, use the appropriate type here
-        // e.g. "Layout".
-        return $this->renderWith(['type' => 'Includes', self::class]);
-    }
-}
-```
```

Comment:
Remove unnecessary example now that the default implementation in `ModelData` does this for you.